### PR TITLE
Properly override host as passed by --source

### PR DESCRIPTION
**Pull request by mvdan**

### CODE REVIEW

**Comment by mvdan** [comment]
Maybe I should have clarified in the commit message that this is for the `kubernetes` source.

File: sources/kube_factory.go [hunk 1]
```diff
@@ -102,6 +102,7 @@ func CreateKubeSources(uri string, options map[string][]string) ([]api.Source, e
 			return nil, err
 		}
 
+		kubeConfig.Host = configOverrides.ClusterInfo.Server
```
Comment (vmarmol) [inline sources/kube_factory.go]: This should probably only override if it was provided no?

Comment (mvdan) [inline sources/kube_factory.go]: Fair point.

**Comment by mvdan** [comment]
@vmarmol just updated the patch with an empty string check. Made sure both the default and overriding cases work.

**Comment by vmarmol** [comment]
LGTM, thanks for being thorough @mvdan!

**Pull request merged**